.github/Dockerfiles/Ubuntu.bionic-non-free.Dockerfile

```diff
@@ -135,7 +134,6 @@ RUN conda update conda -y && \
       pandas==1.0.5 \
       scipy==1.6.3 \
       traits==4.6.0 \
-      wxpython \
       pip && \
     pip install \
       torch==1.2.0 torchvision==0.4.0 -f https://download.pytorch.org/whl/torch_stable.html && \
```

Comment:
Should  these be updated too? (Or moved/merged with requirements.txt)

Reply:
- The `wxpython` GUI was removed in 1.8.1 and isn't needed for any builds based on this Dockerfile.
- Before this PR, this Ubuntu image was/is the base image for both the standard and ABCD-HCP variant (version-matched to [ABCD-HCP BIDS fMRI Pipeline](https://github.com/DCAN-Labs/abcd-hcp-pipeline/blob/e480a8f99534f1b05f37bf44c64827384b69b383/Dockerfile)) C-PAC images; after/in this PR, [`Ubuntu.bionic-non-free.Dockerfile`](https://github.com/FCP-INDI/C-PAC/blob/27b9220a41df3ce75479943f9a6c9608200f9a24/.github/Dockerfiles/Ubuntu.bionic-non-free.Dockerfile) is still the base for the ABCD-HCP variant, but [`Ubuntu.Python3.10-bionic-non-free.Dockerfile`](https://github.com/FCP-INDI/C-PAC/blob/27b9220a41df3ce75479943f9a6c9608200f9a24/.github/Dockerfiles/Ubuntu.Python3.10-bionic-non-free.Dockerfile) is now the base for the standard image.


Whenever we revisit the variant images, though, we should figure out how to better handle those dependencies that differ between the different variant images.